Sources/IntExtensions.swift

```diff
@@ -51,10 +51,25 @@ extension Int {
             return -1; //out of bound
         }
     }
-
 }
 
 extension UInt {
     /// EZSE: Convert UInt to Int
     public var toInt: Int { return Int(self) }
+    
+    /// EZSE: Greatest common divisor of two integers using the Euclid's algorithm.
+    /// Time complexity of this in O(log(n))
+    public static func gcd(_ firstNum:UInt, secondNum:UInt) -> UInt {
```

Comment:
Still do not see the purpose of secondNum label, when we autocomplete this function we have two parameters of type UInt. We could infer that the first if firtstUint, and second is secondUint :)